minemeld/ft/json.py

```diff
@@ -110,6 +117,9 @@ def _build_iterator(self, now):
         if self.username is not None and self.password is not None:
             rkwargs['auth'] = (self.username, self.password)
 
+        if self.header_key is not None and self.header_value is not None:
```

Comment:
I think here it should be:
`if self.headers is not None:`

as self.header_key and self.header_value are no longer defined